src/core/tests/preprocess.cpp

```diff
@@ -2,12 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "common_test_utils/test_assertions.hpp"
+#include "gmock/gmock-matchers.h"
```

Comment:
```suggestion
#include "gmock/gmock.h"
```
or remove if included in `test_assertions.hpp`

Reply:
Removed since it's presented in test_assertions.hpp